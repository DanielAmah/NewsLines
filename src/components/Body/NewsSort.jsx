import React from 'react';
import PropTypes from 'prop-types';

/**
 * @param {any} props
 * @returns
 */

const NewsSort = (props) => {
  const { value, text } = props;
  return <option value={value}>{text}</option>;
};

NewsSort.defaultProps = {
  value: '',
  text: ''
};

NewsSort.propTypes = {
  value: PropTypes.string,
  text: PropTypes.string
};

export default NewsSort;
