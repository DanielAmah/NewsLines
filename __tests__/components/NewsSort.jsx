/* global expect test */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NewsSort from '../../src/components/Body/NewsSort';

describe('NewsSort Component', () => {
  test('should match the NewsSort snapshot', () => {
    const component = shallow(<NewsSort />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
