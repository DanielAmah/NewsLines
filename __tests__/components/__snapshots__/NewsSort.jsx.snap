// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsSort Component should match the NewsSort snapshot 1`] = `
<option
  value=""
/>
`;
